src/storage/auth.ts

```
@@ -0,0 +1,115 @@
+    async login(email: string, password: string): Promise<TokenResponse> {
+      this.loading = true;
+      this.error = null;
+      
+      try {
+        const response = await AuthService.login({ email, password });
+        
+        this.setTokens(response.accessToken, response.refreshToken);
+        this.user = jwtDecode<TokenClaims>(response.accessToken) as unknown as User;
+        
+        return response;
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)